lib/helpers.js

```diff
@@ -56,10 +57,14 @@ exports.prefixedHash = x => {
 
 exports.sha256 = x => crypto.createHash('sha256').update(x).digest('base64');
 
-// generates a `groups * 3` char nonce; output is a multiple of 3 to ease
-// human readability
-exports.generateNonce = async ({groups = 3} = {}) => {
-  const bytes = await randomBytes(groups * 3);
-  const length = ALPHABET.length - 1;
-  return [...bytes].map(b => ALPHABET[Math.floor(b / 255 * length)]).join('');
+exports.generateNonce = async ({entryStyle = 'human'} = {}) => {
+  if(entryStyle === 'human') {
+    const bytes = await randomBytes(9);
+    const length = NUMERIC_DIGITS.length - 1;
+    return [...bytes].map(b => NUMERIC_DIGITS[Math.floor(b / 255 * length)])
+      .join('');
+  }
+  if(entryStyle === 'machine') {
+    return await generateId();
```

Comment:
I think we usually opt for a [fixed length ID](https://github.com/digitalbazaar/bnid#generate-an-id) -- @mattcollier do you have a preference for that here?

Reply:
Made it fixed length for now, will update/change later based on feedback from @mattcollier

Reply:
@mattcollier need your feedback here. Thank you.

Reply:
my love for fixed length, predictable tokens is well known.  Fixed length as you have implemented is my preference.  Thank you!